src/components/ApplicationForm/BasicInfo.js

```diff
@@ -0,0 +1,408 @@
+import schools from '../../containers/Application/data/schools.json'
+import majors from '../../containers/Application/data/majors.json'
+
+const genderOptions = [
+  { value: 'female', label: 'Female' },
+  { value: 'male', label: 'Male' },
+  { value: 'non-binary', label: 'Non-binary' },
+  { value: 'other', label: 'Other' },
+  { value: 'prefer not to say', label: 'Prefer not to say' },
+]
+
+const educationOptions = [
+  { value: 'high school', label: 'High school' },
+  { value: 'undergraduate', label: 'Undergraduate' },
+  { value: 'graduate', label: 'Graduate or Post-Graduate' },
+  { value: 'other', label: 'Other' },
+]
+
+// check w logs
+const graduationOptions = [
+  { value: 2019, label: '2019' },
+  { value: 2020, label: '2020' },
+  { value: 2021, label: '2021' },
+  { value: 2022, label: '2022' },
+  { value: 2023, label: '2023' },
+  { value: 2024, label: '2024' },
+  { value: 2025, label: '2025+' },
+]
+
+// check w logs
+const hackathonOptions = [
+  { value: 0, label: '0' },
+  { value: 1, label: '1' },
+  { value: 2, label: '2' },
+  { value: 3, label: '3' },
+  { value: 4, label: '4+' },
+]
+
+// form part 1
+export default ({ formInputs, onChange }) => (
+  <>
+    <FormSpacing>
+      <CenteredH1>
+        Tell us about yourself!{' '}
+        <span role="img" aria-label="smile">
+          &#128578;
+        </span>
+      </CenteredH1>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 01</QuestionHeading>
+      <H1 size="1.5em">What is your preferred name?</H1>
+      <TextInput
+        placeholder="First Name"
+        value={formInputs.firstName}
+        onChange={e =>
+          onChange({
+            ...formInputs,
+            firstName: e.target.value,
+          })
+        }
+      ></TextInput>
+      <TextInput
+        placeholder="Last Name"
+        value={formInputs.lastName}
+        onChange={e =>
+          onChange({
+            ...formInputs,
+            lastName: e.target.value,
+          })
+        }
+      ></TextInput>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 02</QuestionHeading>
+      <H1 size="1.5em">Which gender do you identify as?</H1>
+      <Dropdown
+        options={genderOptions}
+        placeholder="Gender"
+        isSearchable={false}
+        value={formInputs.gender}
+        onChange={e =>
+          onChange({
+            ...formInputs,
+            gender: e.value,
+          })
+        }
+        isValid
+      ></Dropdown>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 03</QuestionHeading>
+      <H1 size="1.5em">What is your race/ethnicity? (Select all that apply)</H1>
+      <Select
+        type="checkbox"
+        label="Asian"
+        checked={formInputs.ethnicity.asian}
+        onChange={() =>
+          onChange({
+            ...formInputs,
+            ethnicity: { ...formInputs.ethnicity, asian: !formInputs.ethnicity.asian },
+          })
+        }
+      ></Select>
```

Comment:
Can we use an map of options like you did for graduation? Also for the options for the "contribution" question (q11)

Reply:
these are multiselects, how should I map it?

Reply:
wrote this on github idk if the syntax is correct but something like this
```
const ethnicityOptions = {
    asian: "Asian",
    black: "Black or African American"
}
{
Object.entries(ethnicityOptions).map(([ethnicity, label]) => {
    return 
    <Select
        type="checkbox"
        label={label}
        checked={formInputs.ethnicity[ethnicity]}
        onChange={() =>
          onChange({
            ...formInputs,
            ethnicity: { ...formInputs.ethnicity, [ethnicity]: !formInputs.ethnicity[ethnicity] },
          })
        }
      />
}
}
```

Reply:
wouldn't this only make it one select button?